include/meson.build

```diff
@@ -18,4 +18,20 @@ vkd3d_idl = [
   'vkd3d_core_interface.idl',
 ]
 
-vkd3d_header_files = idl_generator.process(vkd3d_idl)
+# idl_generator.process() returns GeneratedList
+# even when called with single item so we process
+# each idl file in a loop generating and installing
+# them to avoid calling process() twice
+vkd3d_header_files = []
+foreach idl_file : vkd3d_idl
+  header_file = idl_generator.process(idl_file)
+  vkd3d_header_files += header_file
+
+  header_name = idl_file.split('.')[0] + '.h'
+  custom_target('install_' + header_name,
```

Comment:
You'll want to gate this behind Linux build. It's confusing to install these headers on a Windows build since you should just use normal D3D headers there. See build.meson:186. Otherwise, it seems okay to install the IDL headers.